test/Microsoft.ML.Tests/Scenarios/Api/Metacomponents.cs

```diff
@@ -41,5 +41,103 @@ public void Metacomponents()
                 var predictor = trainer.Train(new TrainContext(trainRoles));
             }
         }
+
+        /// <summary>
+        /// OVA with calibrator argument
+        /// </summary>
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

I'm sorry, I was not clear on that. Move it to Microsoft.ML.Tests/TrainerEstimators directory. And don't use `New_` prefix